src/wcs2_0/src/main/java/org/geoserver/wcs2_0/response/WCS20GetCapabilitiesTransformer.java

```diff
@@ -673,9 +683,9 @@ private void handleBoundingBox(BoundingBox boundingBox) throws Exception {
         }
 
         private void handleLanguages() {
-            //            start("ows:Languages");
-            //            // TODO
-            //            end("ows:Languages");
+            // start("ows:Languages");
```

Comment:
Lots of comments have been reformatted, but this is not the work of google-format, it happened in a IDE I'm guessing. I could have closed an eye for a few cases, but they are everywhere... can you please back out all of these changes?